mkosi/__init__.py

```diff
@@ -2654,16 +2654,15 @@ def install_arch(state: MkosiState) -> None:
     if state.config.release is not None:
         MkosiPrinter.info("Distribution release specification is not supported for Arch Linux, ignoring.")
 
+    assert state.config.mirror
+
     if state.config.local_mirror:
         server = f"Server = {state.config.local_mirror}"
-    elif state.config.mirror:
+    else:
         if platform.machine() == "aarch64":
```

Comment:
Since this is the mirror for the image we're building, shouldn't this be `config.arch`?

Reply:
Yeah, good point